libr/core/cbin.c

```diff
@@ -2049,61 +2054,65 @@ static int bin_symbols(RCore *r, int mode, ut64 laddr, int va, ut64 at, const ch
 			const char *name = sn.demname? sn.demname: r_symbol_name;
 			r_cons_printf ("%s\n", name);
 		} else if (IS_MODE_RAD (mode)) {
-			RBinFile *binfile;
-			RBinPlugin *plugin;
-			char *name = strdup (sn.demname? sn.demname: r_symbol_name);
-			r_name_filter (name, -1);
-			if (!strncmp (name, "imp.", 4)) {
-				if (lastfs != 'i') {
-					r_cons_printf ("fs imports\n");
-				}
-				lastfs = 'i';
-			} else {
-				if (lastfs != 's') {
-					r_cons_printf ("fs %s\n",
-						exponly? "exports": "symbols");
-				}
-				lastfs = 's';
-			}
-			if (r->bin->prefix) {
-				if (symbol->dup_count) {
-					r_cons_printf ("f %s.sym.%s_%d %u 0x%08"PFMT64x"\n",
-						r->bin->prefix, name, symbol->dup_count, symbol->size, addr);
+			/* Skip special symbols because we do not flag them and
+			 * they shouldn't be printed in the rad format either */
+			if (!is_special_symbol (symbol)) {
```

Comment:
is_special_symbol is checked in another conditional above. so this is doing nothing, i would rather remove the check above and keep it like this, but inverting the conditional to behave properly to set the ahb, Cd..

if (is_special_symbol(symbol)) {
  ...
} else {
...

Reply:
No this is not done above. This is done above only if `IS_MODE_SET (mode)` is true, which means this won't get executed during printing in RAD mode.